src/msgwindow.c

```diff
@@ -83,6 +83,11 @@ enum
 };
 
 
+static GdkColor color_error = {0, 65535, 0, 0};
+static GdkColor color_context = {0, 65535 / 2, 0, 0};
+static GdkColor color_message = {0, 0, 0, 0xD000};
```

Comment:
Oh. I didn't check that one as you had done another "superseding" one by the time I checked anything.  Still applies though :)